src/vivarium_public_health/disease/transition.py

```diff
@@ -14,11 +14,11 @@ def setup(self, builder):
         rate_data, pipeline_name = self._get_rate_data(builder)
         self.base_rate = builder.lookup.build_table(rate_data)
         self.effective_rate = builder.value.register_rate_producer(pipeline_name, source=self.rates)
+        self.base_paf = lambda index: [builder.lookup.build_table(0)(index)]
```

Comment:
why does this need to be a distinct attribute?

Reply:
I don't think it does